beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -98,13 +100,9 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 		traceutil.AnnotateError(span, err)
 		return pubsub.ValidationReject
 	}
-
-	// The attestation's committee index (attestation.data.index) is for the correct subnet.
-	digest, err := s.forkDigest()
-	if err != nil {
-		log.WithError(err).Error("Failed to compute fork digest")
+	if err := s.chain.VerifyFinalizedConsistency(ctx, att.Data.BeaconBlockRoot); err != nil {
```

Comment:
lets perform the finalized consistency check first, as in the event the target root is far beyond our current finalized checkpoint, retrieving from our ancestor through the db is very expensive in `VerifyLmdFfgConsistency` .